packages/core/src/lib/template-engine/built-in-extensions/cache-layer/index.ts

```diff
@@ -0,0 +1,3 @@
+import { CacheTagBuilder } from './cacheTagBuilder';
```

Comment:
I am a little bit confused about the folder name, cause we have another "cache-layer" folder under /core/src/lib
and we have a "query-builder" folder that seems to work the same as the "template-engine/build-in-extension/cache-layer"

Reply:
Maybe rename it to "cache-builder"?

Reply:
Maybe `cache`?

Reply:
I think `cache` is good.